ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -289,6 +294,8 @@ wan_router:
       wan_ha:
         enabled: true # TODO AVD4.8.0: Remove once WAN HA is GA.
         ha_interfaces: [Ethernet52]
+        # Test wan_ha mtu
+        mtu: 9194
```

Comment:
if this is to check precedence then maybe pick a different value than the default one?

Reply:
updated